dev/generate_rule_metadata.py

```diff
@@ -80,6 +83,11 @@ class Rule:
     }
 }
 
+benchmark_to_posture_type = {
```

Comment:
Maybe I'm not updated but I thought we decided to use the `posture_type` propagated as part of the agent config...
Will save us to maintain such maps.

Reply:
I thought about it and this feels more correct to me. `cloudbeat` doesn't touch the `rule` object in the findings at all. @oren-zohar WDYT?

Reply:
That's true for now, but I can think of use in such a field.
For example, determining if a leader election mechanism should be active.

Reply:
I agree with @ari-aviran, cloudbeat enriches the events, but never touches the `rule` part